### ISSUE TITLE
Install KJV example not working

### ISSUE THREAD

**Issue by nikitazdvijkov** [ezra-bible-app/node-sword-interface#23]
Steps:

1. Install dependencies for Mac using instructions in README.
2. Clone and `npm install` the repo.
3. Run `node examples/install_kjv.js`.

Result:

```
Detected old InstallMgr directory installMgr.
Updating repository configuration ...
Installing King James module
/Users/asdf/dev/node-sword-interface/index.js:240
      this.nativeInterface.installModule(moduleCode, progressCB, function(result) {
                           ^

TypeError: String expected for argument 1
    at /Users/asdf/dev/node-sword-interface/index.js:240:28
    at new Promise (<anonymous>)
    at NodeSwordInterface.installModule (/Users/asdf/dev/node-sword-interface/index.js:239:12)
    at installKJV (/Users/asdf/dev/node-sword-interface/examples/install_kjv.js:30:19)
```

**Comment by nikitazdvijkov** [comment]
I get the same error when I attempt to create a new project using `node-sword-interface`. 

Steps:

```
npm init -y
npm install node-sword-interface
```

Copy the example from the README into `install_kjv.js`:

```
const NodeSwordInterface = require('node-sword-interface');
var interface = new NodeSwordInterface();

async function installKJV() {
  console.log("Updating repository configuration ...");
  await interface.updateRepositoryConfig();

  console.log("Installing King James module");
  // Install the King James Version (Uses the internet connection to download and install the module)
  await interface.installModule('KJV');
}

installKJV().then(() => {
  console.log("Installation of KJV successfully completed!");
});
```

Run via `node install_kjv.js`.

**Comment by nikitazdvijkov** [comment]
Node v16.5.0

NPM v7.21.0

**Comment by tobias-klein** [comment]
Thank you so much @nikitazdvijkov for reporting this!
Something must have changed in the API since the example has been created.
This will certainly be fixed in the next version.

**Comment by nikitazdvijkov** [comment]
That fixed it, thanks!

**Comment by tobias-klein** [comment]
I fixed the API with 764237d1998734e53aea4ea7740d37e805c6911b and version 0.236.0 (published on npmjs). The examples are now working again without any changes.

**Comment by kthenrique** [comment]
Oh this is odd! Now it works... I've no clue what is different now! sorry for the inconvenience.
